packages/terra-date-input/src/DateInput.jsx

```diff
@@ -482,6 +484,17 @@ class DateInput extends React.Component {
       { incomplete: (this.props.isIncomplete && this.props.required && !this.props.isInvalid) },
     ]);
 
+    let ariaDescriptionIds;
+    if (useExternalFormatMask === false) {
+      if (this.props.monthAttributes && this.props.monthAttributes['aria-describedby']) {
+        ariaDescriptionIds = `${this.formatDescriptionId} ${this.props.monthAttributes['aria-describedby']}`;
+      } else {
+        ariaDescriptionIds = this.formatDescriptionId;
+      }
+    } else if (this.props.monthAttributes && this.props.monthAttributes['aria-describedby']) {
+      ariaDescriptionIds = this.props.monthAttributes['aria-describedby'];
+    }
+
```

Comment:
i am seeing this  logic is common between all the inputs. can we have common function for this ? so that can be called in these places ?

Reply:
Updated here https://github.com/cerner/terra-framework/pull/1263/commits/3d6cfbdb7477c4c4b51befacd432c4dd5c9e5c37.